BootloaderCommonPkg/Library/Ext23Lib/Ext2FsLs.c

```diff
@@ -214,15 +216,15 @@ Ext2fsLs (
     do {
       New = PNames;
       if (New->EntryType == 2) {
-        DEBUG ((DEBUG_INFO, "  %a/\n", New->EntryName));
+        ConsoleOutFunc (L"  %a/\n", New->EntryName);
       } else if (New->EntryType == 1) {
         FileSize = 0;
         Rc = ReadInode (New->EntryInode, File);
         if (!Rc) {
           Fp = (FILE *)File->FileSystemSpecificData;
           FileSize = Fp->DiskInode.Ext2DInodeSize;
         }
-        DEBUG ((DEBUG_INFO, "  %-16a %d\n", New->EntryName, FileSize));
+        ConsoleOutFunc (L"  %-16a %d\n", New->EntryName, FileSize);
```

Comment:
Better to check if  ConoleOutFunc is NULL since it is passed in from outside.  Similar for line 141 and 219.

Reply:
Maybe have a single check at the function entry.  If ConsoleOutFunc is NULL, return error.

Reply:
Thanks for the comment. You are right. Null checking at the function entry is better way. Let me update it in both Fat and Ext libs.